lib/hexo/post.js

```diff
@@ -260,12 +258,7 @@ class Post {
     }).then(() => {
       data.content = cacheObj.escapeContent(data.content);
 
-      if (isSwig) {
-        // Render with Nunjucks if this is a swig file
```

Comment:
I'm unsure about this change. I cannot figure out if this removal is covered by unit tests.